# Crecimiento del PIB en Colombia (últimos 10 años)

## Datos del crecimiento del PIB en Colombia
A continuacion se muestran los datos simulados del crecimiento anual del PIB en Colombia (%)

In [7]:
# Lista de datos simulados
pib = [4.5, 3.2, 2.8, 1.4, -7.0, 10.6, 7.3, 3.5, 1.2, 0.6]

## Calcular la media del crecimiento

In [8]:
# Calcular la suma de todos los valores
suma = 0
for valor in pib:
    suma += valor

# Se calcula el promedio con el resultado de la suma
media = suma / len(pib)

print ("La media del crecimiento: ", media)

La media del crecimiento:  2.81


El crecimiento promedio del PIB es aproximadamente del 2.81%, lo que señala un crecimiento moderado en el periodo definido

## Determinar el año con mayor crecimiento

In [9]:
# Se inicializa la variable para la lista
maximo = pib[0]

# Se busca el valor maximo entre toda la lista de valores
for valor in pib:
    if valor > maximo:
        maximo = valor

print("Mayor crecimiento:", maximo)

Mayor crecimiento: 10.6


El valor maximo 10.6% refleja una fuerte recuperacion en ese año.

## Identificar el valor minimo

In [10]:
# Se inicializa la variable para la lista
minimo = pib[0]

# Se busca el minimo entre todos los valores
for valor in pib:
    if valor < minimo:
        minimo = valor

print("Valor minimo:", minimo)

Valor minimo: -7.0


El valor minimo de -7.0 representa una caida considerable la cual nos presenta una crisis economica

## ¿Qué efecto tiene el dato negativo en la media?

In [11]:
# Calcular la media
media = sum(pib) / len(pib)

print("Media del PIB:", media)

Media del PIB: 2.8099999999999996


El dato negativo de la media reduce el promedio general al disminuir la suma total de los datos en constraste si el dato fuera positivo

## Determinar los años con crecimiento positivo

In [12]:
# Inicializando la variable de los valores positivos
positivo = 0

# Se buscan los valores mayores que 0
for valor in pib:
    if valor > 0:
        positivo += 1

print("Años con crecimiento positivo:", positivo)

Años con crecimiento positivo: 9


9 de los 10 años del crecimiento pib confirma que a pesar del dato negativo en un año, hubo una tendencia general de crecimiento economico.